x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/entsearch/engine/action/ListEnginesAction.java

```diff
@@ -0,0 +1,168 @@
+/*
```

Comment:
I don't see any unit tests for classes in the `action` package, I'm guessing we don't do that in the ES codebase. Are there any other means of testing their functionality?

Reply:
@demjened we've been doing REST yaml tests and serialization tests. I believe @carlosdelest will be adding these shortly, but they're pretty boilerplate. If you would like to look at an example you can see them in PRs such as [this one](https://github.com/elastic/elasticsearch/pull/94090)

Reply:
Added REST and serialization tests, please check 👍